packages/nitro-protocol/contracts/NitroAdjudicator.sol

```diff
@@ -41,7 +41,7 @@ contract NitroAdjudicator is MultiAssetHolder {
             sigs
         );
 
-        transferAllAssets(channelId, outcomeBytes, bytes32(0), address(0));
+        transferAllAssets(channelId, outcomeBytes, bytes32(0));
```

Comment:
Unrelared to current PR, but stuck out to me as something I didn't know:

`stateHash = 0` is a magic value for concluded channels?

Reply:
Yes. This is buried in the docs https://docs.statechannels.org/docs/protocol-tutorial/release-assets but should be more prominent.

Reply:
I don't think there's any real reason to require `stateHash == 0`, it's just convenient. (Note that setting `stateHash = 0` during the execution of exits is [one idea](https://github.com/statechannels/statechannels/pull/3624#discussion_r677590732) for how to clearly separate adjudication and asset management.)